### PR TITLE
Add get full collection support

### PR DESCRIPTION
**Pull request by zhangchiqing**
To be able to resolve the dependency issue for https://github.com/onflow/flow-go/pull/5910

### CODE REVIEW

File: emulator/pendingBlock.go [hunk 1]
```diff
@@ -126,6 +126,20 @@ func (b *pendingBlock) Collections() []*flowgo.LightCollection {
...
 
+func (b *pendingBlock) FullCollections() []*flowgo.Collection {
+	if len(b.transactionIDs) == 0 {
+		return []*flowgo.Collection{}
+	}
+
+	transactions := make([]*flowgo.TransactionBody, len(b.transactionIDs))
+	for i, txID := range b.transactionIDs {
+		transactions[i] = b.transactions[txID]
+	}
```
Comment (turbolent) [inline emulator/pendingBlock.go:137]: Maybe just use `copy`?

**Review by turbolent — APPROVED** [review]

**Review by peterargue — APPROVED** [review]

File: emulator/pendingBlock.go (hunk 1 above)
Comment (zhangchiqing) [reply emulator/pendingBlock.go:137]: How to `copy`? It's reading a transaction by id from a map.

Comment (peterargue) [reply emulator/pendingBlock.go:137]: I think this approach is needed to preserve the order

Comment (turbolent) [reply emulator/pendingBlock.go:137]: Ah, sorry, I didn't realize the input was a map

**Pull request merged**